src/scss/constants.scss

```diff
@@ -42,6 +42,7 @@ $playa-venue-live: rgb(125, 223, 194);
 $profile-image-bg-color: #999999;
 
 $z-layers: (
+  nav-search-results: 380,
```

Comment:
This is the same `z-index` value that was already being used, just moved to our centralised helper. I didn't want to  expand the risk of this PR by changing it here. At some point we should align all of our `z-index` values in this position, but probably with 'larger gaps' between them (like each 'new layer' would be 10 or 100 different from the last, then minor tweaks made in the digits between)